src/client.rs

```diff
@@ -0,0 +1,84 @@
+use std::fs;
+
+use crate::auth::SovaAuth;
+use crate::pem::TESTNET_CA_PEM;
+use crate::proto::auth::Token;
+use crate::searcher::SovaSearcher;
+
+struct SovaClient {
```

Comment:
```suggestion
pub struct SovaClient {
```
`SovaClient` struct must be public